common/models/user.js

```diff
@@ -7,6 +7,41 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+
+//send email again incase of unverified email
+User.triggerEmail = function(userId, cb) {
+  User.find({where: {id: userId}}, function(err, user)
+  {
+    const userObj = user[0];
+      var options = {
+      type: 'email',
+      to: userObj.email,
+      from: 'noreply@loopback.com',
+      subject: 'Thanks for registering.',
+      template: path.resolve(__dirname, '../../server/views/verify.ejs'),
+      redirect: '/verified',
+      user: userObj
+    };
+    userObj.verify(options, function(err, response) {
+      if (err) {
+        User.deleteById(userObj.id);
+        return next(err);
+      }
+      console.log('> verification email sent:', response);
+    });
+  })
+  cb(null, 'email triggered');
```

Comment:
I need to know a way here to invoke UI page after email is triggered instead of a callback